File: src/gestorAplicacion/clasesPrincipales/Ejecutivo.java
package gestorAplicacion.clasesPrincipales;
import java.util.ArrayList;

public class Ejecutivo extends Bus {
	private int capacidad;
	private final int precio=500;//precio por km
	
	//constructor
	public Ejecutivo(String modelo, String placa, Conductor conductor, int capacidad, ArrayList<String> sillas) {
		super(modelo,placa,conductor,sillas);
		this.capacidad=capacidad;
		this.codigo=150;
	}
	
	//get&set capacidad
	public int getCapacidad() {
		return capacidad;
	}
	public void setCapacidad(int capacidad) {
		this.capacidad=capacidad;
	}
	//get_precio
	public int getPrecio() {
		return precio;
	}
	@Override
	public void sillasDisponibles() {
		for(Tiquete tiquete : Viaje.gettiquetesTodos()){
			if(tiquete.getEstado() == false){
				System.out.println("la silla: "+tiquete.getSillaTiquete()+"esta disponible");
				}
			}
		}
	public String sillaTiquete(Tiquete tiquete) {
		return tiquete.getSillaTiquete();
	}
	//public String getSillaTiquete() {
		//return sillaTiquete;}
}

File: Python/gestor_aplicacion/clasesHerencia/Ejecutivo.py
from gestor_aplicacion.clasesPrincipales.Bus import Bus
from gestor_aplicacion.clasesPrincipales.Viaje import Viaje


class Ejecutivo(Bus):
    _ejecutivos = []

    def __init__(self, placa, conductor, sillas):
        # instance fields found by Java to Python Converter:
        self._precioKm = 500
        self._capacidad = 24
        self.name = "Ejecutivo"
        self.placa = placa
        self.conductor = conductor
        super().__init__(placa, conductor, sillas)
        self.codigo = 150
        Ejecutivo._ejecutivos.append(self)


    def getPlaca(self):
        return self.placa

    def getConductor(self):
        return self.conductor

    # get&set capacidad

    def getCapacidad(self):
        return self._capacidad

    # get_precio
    def getPrecioKm(self):
        return self._precioKm
    # get&set codigo

    def getCodigo(self):
        return self.codigo

    def setCodigo(self, codigo):
        self.codigo = codigo

    def sillasDisponibles(self):
        for tiquete in Viaje.getTiquetesTodos():
            if tiquete.getEstado() == False:
                print("la silla: "+tiquete.getSillaTiquete()+"esta disponible")

    def sillaTiquete(self, tiquete):
        return tiquete.getSillaTiquete()
